.env.dev

```
@@ -11,8 +11,8 @@ DB_CONNECTION=mysql
 DB_HOST=mysql
 DB_PORT=3306
 DB_DATABASE=monica
-DB_USERNAME=homestead
-DB_PASSWORD=secret
+DB_USERNAME=root
```

Please don't change this
```suggestion
DB_USERNAME=homestead
```